### ISSUE TITLE
unloadable or broken datasets

### ISSUE THREAD

**Issue by neutrinoceros** [yt-project/website#104]
We have identified at least 3 datasets from https://yt-project.org/data/ that seem impossible to load (either using `yt.load` or `yt.load_sample`):
- ramses_extra_fields (targeted for removal in #102, but could in fact be fixed by repackaging the tarball according to @cphyc)
- output_00080_halos : I couldn't find a working way to load this one (apparently this is a very rare case of a two-stages load ?)
- CompactObjects: this one isn't used anywhere and seems to be unloadable currently (fails either with `YTUnidentifiedDataFormat` if passed the base directory or a segfault if passed any of the embedded files). I wonder wether it can be salvaged.

**Comment by cphyc** [comment]
For `output_00080_halos`, you first need to load `output_00080` which you then feed as follows
```python
ds = yt.load("output_00080")
halos = yt.load(output_00080_halos, parent_ds=ds)
```
I don't see a simple way to support it via the `load_sample` mechanism.

**Comment by neutrinoceros** [comment]
Indeed. I think it warrants a creating a blacklist. The better solution would be to make it loadable in a single step (with kwargs ?), though right now I'm not 100% convinced it's feasible.
